src/parser-browsers.js

```diff
@@ -590,9 +590,10 @@ const browsersList = [
   {
     test: [/.*/i],
     describe(ua) {
+      const regexp = ua.search('\\(') === -1 ? /^(.*)\/(.*) / : /^(.*)\/(.*)[ \t]\((.*)/;
```

Comment:
Could you, please, leave a comment in the code with some details on why exactly we use this `ua.search('\\(') === -1` here? :)

Reply:
Yes, sure, I can definitely do that! The idea is to use the search on the string to check if it has the "(" char, in case there is a device information in the User Agent, otherwise use the old separator. Is it ok to put it like that or should I be more technical?

Reply:
I'd suggest creating a variable like `const hasDeviceSpec = ua.search('\\(') !== -1;` and put a comment for it, that here we try to make sure that there are explicit details about the device in order to decide what regexp exactly we want to apply (as there is a specific decision based on that conclusion). It would make the code more understandable in the future :) Thanks!

Reply:
hey @lancedikson, sorry for the delay, but I have made the changes, what do you think?